examples/forms.amp.html

```diff
@@ -72,7 +86,8 @@
 <form method="post"
       action="/form/html/post"
       action-xhr="/form/echo-json/post"
-      target="_blank">
+      target="_blank"
+      on="submit:lightbox1">
```

Comment:
I just figured no reason to listen to that event if the users do haven't included the `amp-form`. But I guess we're already kinda doing that with `document-submit` so might as well add it for the events right away. 

I was also worried about the sequence in which events handlers will be called, my understanding if the other ones are capture they should be called first, no?